<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selection</class>
 <widget class="QDialog" name="Selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="player1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Player 1</string>
     </property>
     <widget class="QRadioButton" name="p1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>小新</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="p2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string> 小葵</string>
      </property>
     </widget>
     <widget class="QLabel" name="p1_img">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="p2_img">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="player2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Player 2</string>
     </property>
     <widget class="QRadioButton" name="p3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string> 美冴</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="p4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string> 廣志</string>
      </property>
     </widget>
     <widget class="QLabel" name="p3_img">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="p4_img">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="gameBackground">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>541</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Background</string>
     </property>
     <widget class="QRadioButton" name="b1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> beach</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="b2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> sea</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="b3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> space</string>
      </property>
     </widget>
     <widget class="QLabel" name="b1_img">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>130</y>
        <width>120</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="b2_img">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>120</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="b3_img">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>120</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ready">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
